pkg/frontend/adminactions/resources_list.go

```diff
@@ -76,3 +65,39 @@ func (a *adminactions) ResourcesList(ctx context.Context) ([]byte, error) {
 
 	return json.Marshal(armResources)
 }
+
+func (a *adminactions) appendAzureNetworkResources(ctx context.Context, armResources []arm.Resource, vNetID string, vnetClient network.VirtualNetworksClient, routeTablesClient network.RouteTablesClient) ([]arm.Resource, error) {
+	r, err := azure.ParseResourceID(vNetID)
+	if err != nil {
+		return armResources, err
+	}
+
+	vnet, err := vnetClient.Get(ctx, r.ResourceGroup, r.ResourceName, "")
+	if err != nil {
+		return armResources, err
+	}
+	armResources = append(armResources, arm.Resource{
+		Resource: vnet,
+	})
+	if vnet.Subnets != nil {
+		for _, snet := range *vnet.Subnets {
```

Comment:
it would make sense only to look at the master and worker subnets - these resource IDs for these are in the cluster model

Reply:
Added filtering. I was considering getting the Subnets by ID via the API, but we already have all the information locally in the VNet resource.